src/test/java/com/exasol/adapter/dialects/AbstractSqlDialectTest.java

```diff
@@ -177,4 +183,50 @@ void testGetStringLiteralWithNull() {
         final SqlDialect sqlDialect = new DummySqlDialect(null, AdapterProperties.emptyProperties());
         assertThat(sqlDialect.getStringLiteral(null), equalTo("NULL"));
     }
+
+    @Test
+    void testOmitParenthesesFalse() {
+        final SqlDialect sqlDialect = new DummySqlDialect(null, AdapterProperties.emptyProperties());
+        assertThat(sqlDialect.omitParentheses(ScalarFunction.ADD_DAYS), equalTo(false));
+    }
+
+    @Test
+    void testGetTableCatalogAndSchemaSeparator() {
+        final SqlDialect sqlDialect = new DummySqlDialect(null, AdapterProperties.emptyProperties());
+        assertThat(sqlDialect.getTableCatalogAndSchemaSeparator(), equalTo("."));
+    }
+
+    @Test
+    void testGetSqlGenerationVisitor() {
+        final SqlDialect sqlDialect = new DummySqlDialect(null, AdapterProperties.emptyProperties());
+        SqlGenerationContext context = new SqlGenerationContext("catalogName", "schemaName", false);
+        assertThat(sqlDialect.getSqlGenerationVisitor(context), instanceOf(SqlGenerationVisitor.class));
+    }
+
+    @Test
+    void testGetScalarFunctionAliases() {
+        final SqlDialect sqlDialect = new DummySqlDialect(null, AdapterProperties.emptyProperties());
+        assertThat(sqlDialect.getScalarFunctionAliases(), equalTo(new EnumMap<>(ScalarFunction.class)));
```

Comment:
```suggestion
        assertThat(sqlDialect.getScalarFunctionAliases(), emptyCollectionOf(ScalarFunction.class));
```

See: http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/collection/IsEmptyCollection.html

Reply:
done